app/components/recognition/form.hbs

```diff
@@ -65,29 +70,56 @@
                       <label
                         class="au-c-card-item__label au-c-card-item__label--data"
                       >namelijk</label>
-                      <AuInput
-                        @value={{mut
-                          this.currentRecognition.recognitionModel.awardedBy
-                        }}
-                        id="awardedBy"
-                        @width={{"block"}}
-                      />
+                      <div class="au-u-flex au-u-flex--column au-u-1-1">
+                        <AuInput
+                          @value={{mut model.awardedBy}}
+                          {{on "input" (fn this.clearFormError "awardedBy")}}
+                          id="awardedBy"
+                          @error={{this.validationErrors.awardedBy}}
+                          @width={{"block"}}
+                        />
+                        <AuHelpText
+                          @error={{this.validationErrors.awardedBy}}
+                        >{{this.validationErrors.awardedBy}}</AuHelpText>
+                      </div>
                     </section>
+
                   {{/if}}
                 </:content>
               </Item>
             </:left>
             <:right as |Item|>
               <Item>
-                <:label>Einddatum</:label>
+                <:label>
+                  <AuLabel />
+                </:label>
+                <:content>
+                  {{! TODO: find better way to fix this? }}
+                  {{! template-lint-disable no-inline-styles }}
```

Comment:
better to create a class for this :)

Reply:
Better yet https://appuniversum.github.io/ember-appuniversum/?path=/story/utilities-hide--page

Reply:
For some reason the utility class hide--page does not do what we want (just taking up the place of an input field), so I created a class.